app/src/components/popups/StopPopup.vue

```diff
@@ -152,49 +153,37 @@ const augmentedArrivals = computed<(Omit<StopArrival, 'eta'> & { nextStopName?:
       return null;
     }
 
-    return stop.value.arrivals
-      .toSorted((a, b) => {
-        if (a.eta === 0 || b.eta === 0) {
-          return a.planned.localeCompare(b.planned);
-        }
-        return a.eta - b.eta;
-      })
-      .map((a) => {
-        const trip = trips.value[a.tripId];
-
-        let nextStopName: string | undefined;
-        if (trip !== undefined && trip.arrivals !== undefined) {
-          const nextStopIndex = trip.arrivals.findIndex((s) => s.id === props.marker.id);
-          if (nextStopIndex !== -1) {
-            nextStopName = trip.arrivals[nextStopIndex + 1]?.name;
+    return (
+      stop.value.arrivals
+        .toSorted((a, b) => {
+          if (a.eta === 0 || b.eta === 0) {
+            return a.planned.localeCompare(b.planned);
           }
-        }
-
-        return {
-          ...a,
-          nextStopName,
-          eta: eta(a),
-        };
-      });
+          return a.eta - b.eta;
+        })
+        // eslint-disable-next-line arrow-body-style
+        .map((a) => {
+          // const trip = trips.value[a.tripId];
+
+          // let nextStopName: string | undefined;
+          // if (trip !== undefined && trip.arrivals !== undefined) {
+          //   const nextStopIndex = trip.arrivals.findIndex((s) => s.id === props.marker.id);
+          //   if (nextStopIndex !== -1) {
+          //     nextStopName = trip.arrivals[nextStopIndex + 1]?.name;
+          //   }
+          // }
+
+          return {
+            ...a,
+            nextStopName: undefined,
+            eta: eta(a),
+          };
+        })
+    );
   },
 );
 
-watch(
-  marker,
-  async (newMarker, oldMarker) => {
-    if (newMarker.id === oldMarker?.id) {
-      return;
-    }
-    if (subject !== null) {
-      void unsubscribe(subject);
-    }
-    subject = `data.map.stop.${newMarker.id}`;
-    await subscribe(subject, stops);
-  },
-  { immediate: true },
-);
-
-const tripSubscriptions = new Set<string>();
+// const tripSubscriptions = new Set<string>();
```

Comment:
@anbraten Is this no longer needed?

Reply:
Also part of the nextStop feature which we disabled.